optimum/onnxruntime/quantization.py

```diff
@@ -195,32 +209,32 @@ def partial_fit(
         self,
         dataset: Dataset,
         calibration_config: CalibrationConfig,
-        onnx_augmented_model_name: str = "augmented_model.onnx",
+        onnx_augmented_model_name: Union[str, Path] = "augmented_model.onnx",
         operators_to_quantize: Optional[List[NodeType]] = None,
         batch_size: int = 1,
         use_external_data_format: bool = False,
         use_gpu: bool = False,
         force_symmetric_range: bool = False,
     ):
         """
-        Perform the calibration step and collect the quantization ranges.
+        Performs the calibration step and collect the quantization ranges.
 
         Args:
             dataset (`Dataset`):
                 The dataset to use when performing the calibration step.
             calibration_config (`CalibrationConfig`):
                 The configuration containing the parameters related to the calibration step.
-            onnx_augmented_model_name (`Union[str, os.PathLike]`):
+            onnx_augmented_model_name (`Union[str, Path]`, *optional*, defaults to `"augmented_model.onnx"`):
                 The path used to save the augmented model used to collect the quantization ranges.
-            operators_to_quantize (`list`, *optional*):
+            operators_to_quantize (`Optional[List[NodeType]]`, *optional*):
                 List of the operators types to quantize.
-            batch_size (`int`, defaults to 1):
+            batch_size (`int`, *optional*, defaults to 1):
```

Comment:
I'm thought we aligned on not using the `*optional*`, is that not? https://huggingface.slack.com/archives/C02P0559X9S/p1669628754896019

Reply:
Yes, I will make a PR for this later (to handle everything)

Reply:
all good